### DISCUSSION TITLE
Add display on start up?

### DISCUSSION THREAD

**Discussion by Everwalker** [nomi-san/parsec-vdd#20] [Ideas]
Will it be possible to automatically add display on startup? Will be useful for headless computers.

**Comment by nomi-san** [comment]
![image](https://github.com/nomi-san/parsec-vdd/assets/38210249/fa8118a7-8429-4b26-8771-07d0d74eee15)
There's an option to a add virtual display when no physical monitor detected.
But I'm not sure it works properly on some machines.

About adding on startup and restore added displays, see #19 #17.

**Reply by Everwalker** [reply]
Thanks. On my computer, if it is not connected to a physical display, vdd won't run on start up. The fallback display also does not work when I unplug the physical display.

**Reply by sillyxp** [reply]
Mine,either. No virture screen will be added on startup when I unplug all physical displays (already select "fallback display")

My PC: AMD R5 5600+AMD 6600

**Reply by nomi-san** [reply]
Let me recheck, it works on my PC with APU, however sometimes external GPU could be detected as single display even without connected physical monitor.

**Reply by wesgurn** [reply]
Any update on this? 

Everything works great, but still have to manually Add Display each time after startup.

**Reply by Etachi2016** [reply]
any solutions on this? could not use it on my PC without physical display
